app/router.js

```diff
@@ -15,7 +14,9 @@ Router.map(function () {
     this.route('switch');
   });
   this.route('login');
-  this.route('mock-login');
+  if (config.environment === 'development') {
```

Comment:
I don't think this will work, as dev server environment is production and needs a mock login

Reply:
@lagartoverde Is it ok if I use this? 
`  if (config.acmidm.clientId === '{{OAUTH_API_KEY}}') {
    this.route('mock-login');
  }`